### ISSUE TITLE
Custom date formatter is ignored when struct conforms to WrapCustomizable

### ISSUE THREAD

**Issue by nubbel** [JohnSundell/Wrap#10]
Hey,

I love your libs Wrap and Unbox - simple and powerful!

Consider this test case:

```swift
	func testDatePropertyWithCustomizableStruct() {
		let date = NSDate()
		
		struct Model: WrapCustomizable {
			let date: NSDate
		}
		
		let dateFormatter = NSDateFormatter()
		
		do {
			let model = Model(date: date)
			
			try VerifyDictionary(Wrap(model, dateFormatter: dateFormatter), againstDictionary: [
				"date" : dateFormatter.stringFromDate(date)
				])
		} catch {
			XCTFail(error.toString())
		}
	}
```

It does fail, because the custom date formatter is not passed to the `Wrapper` that wraps the customisable struct. I guess, we must change the signature of the `WrapCustomizable.wrap` method, such that it accepts a date formatter object. But maybe you could think of something more clever.

**Comment by nubbel** [comment]
In case someone runs into this issue, this is my current workaround:

```swift
	func wrapPropertyNamed(propertyName: String, withValue value: Any) throws -> AnyObject? {
		switch propertyName {
		case "lastUpdate":
			return RFC3339NanoDateFormatter.stringFromDate(lastUpdate)
		default:
			return nil
		}
	}
``swift`

**Comment by JohnSundell** [comment]
@nubbel Extremely sorry about the long wait for fixing this, but it has now been fixed in Wrap 2.0, which is about to be released: https://github.com/JohnSundell/Wrap/pull/17

**Comment by nubbel** [comment]
@JohnSundell Awesome, thank you!
